### PR TITLE
aggregation keys doc

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request: create aggregation keys doc from https://docs.google.com/document/d/1bU0a_njpDcRd9vDR0AJjwJjrf3Or8vAzyfuK8JZDEfo/view

- https://pr-7116-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/aggregation-keys/
-
-

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 1]
```diff
@@ -0,0 +1,1373 @@
...
+
+ADD CODE BACK IN
+
+In practice, summary reports are encoded in a way that will make buckets and values look different than stated above (that is, a bucket may look like `\u0000\u0000\x80\u0000`). <em>Bucket</em> and <em>value</em> are both bytestrings. 
+
+### Aggregation keys in practice
+
+Aggregation keys are defined by an adtech company, typically in two steps: when an ad is clicked or viewed, and when a user converts.
+
+{% Aside }
```
Comment (tunetheweb) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:226]: This is your problem:

```suggestion
{% Aside %}
```

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md]: thanks Barry. completely missed it!

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 2]
```diff
@@ -0,0 +1,1001 @@
...
+  - maudn
+---
+
+
+As an ad tech company running campaigns in multiple locations for various product categories, you want to help advertisers answer the following questions:
+
+1.  How many **purchases** of each product category did each of my campaigns in each geographic region generate?
+1.  How much **revenue** for each product category did each of my campaigns in each geographic region generate?
+
+While many ad tech companies encourage advertisers to configure a variety of conversion types, focusing on the most important conversions such as purchases is a good way to ensure that summary results are detailed and accurate for these important events.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:20]: Which important event? I'm a little confused with the logic of this sentence.

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 3]
```diff
@@ -0,0 +1,1001 @@
...
+
+## Dimensions, keys, and values
+
+To answer these questions, let's take a look at dimensions, keys, and values.
+
+### Dimensions
+
+To understand how your campaigns are generating revenue, as described here, you'll want to track the following dimensions:
+
+- Ad campaign ID
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:32]: From a purely style guide perspective, I'd expect all of these to be uniform, ie, 
Dimension name: definition
or
Just dimension name.

Not a hybrid version with both.

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 4]
```diff
@@ -0,0 +1,1001 @@
...
+
+While the Campaign ID and the Geography ID dimensions are known when the ad is served (ad-serving time), the Product category will be known from a trigger event, when the user completes a conversion (conversion time).
+
+{% Aside %}
+For the diagram in this document, we'll display dimensions as follows:
+- Dimensions known at ad-serving time on a green background.
+- Dimensions known at conversion time on a yellow background and underlined.
+{% endAside %}
+
+The dimensions you want to track for this example look like this:
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:44]:
```suggestion
The dimensions you want to track for this example are as shown in the following image:
```

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 5]
```diff
@@ -0,0 +1,1001 @@
...
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/7H0FNCjZM9zr3VE48Q2G.png", alt="Campaign ID, Geography ID, and product category.", width="498", height="55" %}
+
+### What are aggregation keys (buckets)?
+
+Aggregation key and bucket refer to the same thing. The term _aggregation key_ is used in the browser APIs used to configure reports. The term _bucket_ is used in the aggregatable and summary reports, and in the [aggregation service APIs](/docs/privacy-sandbox/aggregation-service/).
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:49]:
```suggestion
The terms aggregation key and bucket refer to the same thing.  _Aggregation key_ is used in the browser APIs used to configure reports. The term _bucket_ is used in the aggregatable and summary reports, and in the [aggregation service APIs](/docs/privacy-sandbox/aggregation-service/).
```

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 6]
```diff
@@ -0,0 +1,1001 @@
...
+
+An aggregation key (key for short) is a piece of data that represents the values of the dimensions being tracked. Data is later aggregated along each aggregation key.
+
+For example, let's assume you're tracking the dimensions Product category, Geography ID, and Campaign ID.
+
+When a user located in Geography ID 7 sees an ad for Campaign ID 12, and later converts by purchasing a product in Product category 25, you may set an aggregation key that looks like this:
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:55]:
```suggestion
When a user located in Geography ID 7 sees an ad for Campaign ID 12, and later converts by purchasing a product in Product category 25, you may set an aggregation key like the one in the following image:
```

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 7]
```diff
@@ -0,0 +1,1001 @@
...
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/dmpyjw7KIcunQemDqs8i.png", alt="Aggregation key for a conversion.", width="495", height="125" %}
+
+You'll see later that an aggregation key does not look exactly like this in practice, but for now let's focus on the information contained in the key.
+
+### What are aggregatable values?
+
+To answer your questions with the dimensions we've outlined, you want to know, across many users and many keys:
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:63]: I'm confused by this sentence.

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md [hunk 8]
```diff
@@ -0,0 +1,1001 @@
...
+			<td><strong>Purchase count</strong></td>
+		</tr>
+		<tr>
+			<td>How much <strong>revenue</strong>…</td>
+			<td><strong>Purchase value</strong></td>
+		</tr>
+	</tbody>
+</table>
+
+When a user located in Geography ID 7 sees an ad for Campaign ID 12, and later converts by purchasing a product of Product category 25 for $120, you may set an aggregation key and aggregatable values that look like these:
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md:89]: I actually don't know what we do with currency symbols, but is this written assuming only US ad tech?

File: site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md (hunk 7 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/aggregation-keys/index.md]: fixed

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**